dev_tools/docker-compose.yml

```diff
@@ -67,23 +67,33 @@ services:
     ports:
       - 9201:9201
   db:
+    depends_on:
+      - tlsFixer
     container_name: db
+    command: -c ssl=on -c ssl_cert_file=/data/certs/s3.pem -c ssl_key_file=/data/certs/s3-key.pem -c ssl_ca_file=/data/certs/ca.pem
     environment:
       - POSTGRES_PASSWORD=postgres
-      - PGDATA=/var/lib/postgresql/data/pgdata
+      - PGDATA=/data/pgdata
       - POSTGRES_DB=test
     healthcheck:
       test: ["CMD", "pg_isready", "-h", "localhost", "-U", "postgres"]
       interval: 5s
       timeout: 20s
       retries: 3
+    restart: always
     image: postgres:11.2-alpine
     ports:
       - "5432:5432"
     volumes:
-      - pgData:/var/lib/postgresql/data
-      - ./certs:/certs
+      - pgData:/data
       - ./sql_files:/docker-entrypoint-initdb.d
+  tlsFixer:
```

Comment:
What is this and why is it needed?

Reply:
Because postgres requires the certificates to have a mode 600 or 640 depending on of they are owned by the postgres user or root.